fix_includes_test.py

```diff
@@ -1197,6 +1193,40 @@ class Bar;  // lines 5-5
                                  infile})
     self.ProcessAndTest(iwyu_output)
 
+  def testAddForwardDeclareInsideNamespaceWithCompactEndings(self):
+    """Tests we put fwd-decls inside an ns when using compact namespace endings."""
+    infile = """\
+// Copyright 2010
+
+namespace ns { namespace  ns1 { namespace ns2 {
+class Ns2Bang;
+}} // namespace ns2 // namespace ns1
+///+class NsBar;
+class NsBaz;
+                                      ///-
+namespace ns3 { namespace ns4 {       ///-
+class Ns4Bye;                         ///-
+}} // namespace ns4 // namespace ns3  ///-
+}  // namespace ns
+
+int main() { return 0; }
+"""
+    iwyu_output = """\
+add_fwd_decl_inside_namespace_without_compact_endings should add these lines:
+namespace ns { class NsBar; }
+
+add_fwd_decl_inside_namespace_without_compact_endings should remove these lines:
+- namespace ns { namespace ns1 { namespace ns2 { namespace ns3 { namespace ns4 { class Ns4Bye; } } } } } // lines 9-9
+
+The full include-list for add_fwd_decl_inside_namespace_without_compact_endings:
+namespace ns { namespace ns1 { namespace ns2 { class Ns2Bang; } } } // lines 4-4
+namespace ns { class NsBaz; } // lines 6-6
+---
+"""
+    self.RegisterFileContents({'add_fwd_decl_inside_namespace_without_compact_endings':
+                                 infile})
```

Comment:
Indentation looks off here

Reply:
I see now the cases that are misformatted on `master`. I'll update those and push, so you can rebase on something correct.

Reply:
Done.